### PR TITLE
Added air quality data in the environment section

### PR DESCRIPTION
**Pull request by ratchakr**
Added airvisual API that can be used to fetch air quality data in the environment section of the curated list of public API-s.

### CODE REVIEW

**Comment by mikestreety** [comment]
Hi @ratchakr. Thanks for this awesome pull request!

It seems some extra lines have sneaked in to the tables and have broken them however. I'll comment on where they are. If you wouldn't mind removing them, that would be great. Thanks!

File: README.md [hunk 1]
```diff
@@ -215,6 +215,10 @@ Please note a passing build status indicates all listed APIs are available since
 
 | API | Description | Auth | HTTPS | Link |
 |---|---|---|---|---|
+
+| PM2.5.in | PM2.5 Data of China | `apiKey` query string | No | [Go!](http://www.pm25.in/api_doc) |
+| AirVisual | Air quality and weather data | `apiKey` query string | No | [Go!](https://airvisual.com/api) |
+
```
Comment (mikestreety) [inline README.md]: And also this one - thanks!

File: README.md [hunk 2]
```diff
@@ -215,6 +215,10 @@ Please note a passing build status indicates all listed APIs are available since
 
 | API | Description | Auth | HTTPS | Link |
 |---|---|---|---|---|
+
```
Comment (mikestreety) [inline README.md]: Could you remove this new line please?

**Comment by mikestreety** [comment]
Thanks!

**Comment by ratchakr** [comment]
Thanks @mikestreety! Does it look OK now?

**Pull request merged**